### PR TITLE
Create a valid target triple in the LLVM Module.

### PR DESCRIPTION
**Pull request by nlewycky**

### CODE REVIEW

File: lib/llvm-backend/Cargo.toml [hunk 1]
```diff
@@ -19,13 +19,13 @@ byteorder = "1"
...
 git = "https://github.com/TheDan64/inkwell"
-rev = "af4cf4efbb27cdea8a54175ffc18ffd91964618c"
+rev = "5610ee7e1fea4f177d1cd70516a988df286975d8"
 default-features = false
 features = ["llvm8-0", "target-x86"]
 
 [target.'cfg(target_arch = "aarch64")'.dependencies.inkwell]
 git = "https://github.com/TheDan64/inkwell"
-rev = "af4cf4efbb27cdea8a54175ffc18ffd91964618c"
+rev = "5610ee7e1fea4f177d1cd70516a988df286975d8"
```
Comment (MarkMcCaskey) [inline lib/llvm-backend/Cargo.toml]: You can use the `inkwell` on crates.io now, it should be the same as master on this commit and that sets us up to be able to publish the llvm-backend next release!

Comment (nlewycky) [reply lib/llvm-backend/Cargo.toml]: AWWW YEAAHH!!

**Comment by nlewycky** [comment]
bors retry

**Pull request merged**